config/routes.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  root 'root#index'
+end
```

Comment:
- [ ] [REQUIRED] The root route is fine but also kindly create a route that will allow the API endpoint for returning a random greeting. The corresponding action for this route should be placed in the greetings controller.
Example:
``` ruby
namespace :api do
    resources :greetings
end
```
- The namespace :api sets up a namespace for the routes. Namespacing is a way to group related routes under a common prefix (in this case, "api"). This can be useful to organize routes for different parts of your application, such as separating API routes from regular web application routes.

- The resources `:greetings` defines RESTful routes for a resource named "greetings" within the "api" namespace. RESTful routes provide a standard set of actions for working with resources (like create, read, update, delete). The resources method generates these routes based on the specified resource name.

The standard RESTful routes generated by resources :greetings would look something like this:

GET /api/greetings (index action)
GET /api/greetings/:id (show action)
POST /api/greetings (create action)
PUT/PATCH /api/greetings/:id (update action)
DELETE /api/greetings/:id (destroy action)